tutorials/toolkits/axes_grid.py

```diff
@@ -76,7 +76,7 @@
 `.AxesDivider.append_axes` can then be used to create a new axes on a given
 side ("left", "right", "top", "bottom") of the original axes.
 
-colorbar whose height (or width) in sync with the main axes
+colorbar whose height (or width) is in sync with the main axes
 -----------------------------------------------------------
```

Comment:
The underline should match the heading text.

Reply:
Done